core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -42,12 +42,7 @@ THE SOFTWARE.
     <l:tasks>
       <st:include page="tasks-top.jelly" it="${it.owner}" optional="true" />
 
-      <st:include page="tasks-new.jelly" it="${it.owner}" optional="true">
-        <!-- made overridable for ViewGroup that doesn't allow modifications -->
```

Comment:
oh, my bad. I missed it. I updated the file so the only change is the `href="#modal"`